doc/rados/troubleshooting/troubleshooting-pg.rst

```diff
@@ -185,6 +185,53 @@ To instruct Ceph to continue anyway::
 Recovery will proceed.
 
 
+.. _failures-pg-incomplete:
+
+Placement Group Incomplete - Peering Blocked
+============================================
+
+History Les
+-----------
+
+When CEPH shows that inactive PG is in incomplete state::
+
+     ceph pg dump_stuck inactive
+
+And PG peer peering is blocked by history les bound::
+
+     ceph pg <pgid> query
+
+.. code-block:: javascript
+
+ "peering_blocked_by_detail": [
+    {
+        "detail": "peering_blocked_by_history_les_bound"
+    }
+  ]
+
+That mean that data in this PG is already lost.
+And if no OSD left that can rise and heal PG.
+Then the way to recover this is to force recovery to ignore this fact and go with whatever data 
```

Comment:
only way? (cc @tchaikov )

Reply:
As I understand this means that (for 3x replication in my case) all OSDs that contained data from this PG are not in. And in this case if all OSDs that was possible to bring into cluster are already there - this means that data from this PG was lost. 
This can happen, for example in case of 3x replica, if you drop 2 failure domain while recovery was not finished.

Reply:
@agutikov i strived to understand how we we are using history.les when choosing a shard when recovering a PG, but ended up finding that it's difficult to put the criteria in an end-user understandable way. please reference https://github.com/ceph/ceph/blob/master/doc/dev/osd_internals/last_epoch_started.rst for more details.

so, unless we are able to understand the cause of this problem, reproduce it, and address it using the documented approach here, i'm inclined to do not add it in the user-facing document.